.github/workflows/lint.yaml

```diff
@@ -0,0 +1,21 @@
+name: ci
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths-ignore:
+    - '**/*.md'
+jobs:
+  unit:
```

Comment:
Copy/paste error: you'll need a unique job identifier.

Reply:
Yep, good catch. Fixing.